### DISCUSSION TITLE
Guess films

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1057] [Galaxy Trivia] [answered]
👨👨‍🦳🚗⌚
👨‍💼👨‍💼🖊️👽
🐢🐀👊🍕

**Comment by peterbaccaro** [comment]
1. BACK TO THE FUTURE
2. PAUL
3. TEENAGE NINJA MUTANT TURTLES

**Comment by thuongtruong1009** [comment, marked as answer]
Based on the emojis provided, here are my guesses for the films:

1. 👨👨‍🦳🚗⌚: "Back to the Future" - This classic sci-fi adventure film features two main characters (👨👨) including an old scientist (👨‍🦳), a time-traveling car (🚗), and a crucial plot element of a time-traveling watch (⌚).

2. 👨‍💼👨‍💼🖊️👽: "Men in Black" - This sci-fi comedy film involves two agents (👨‍💼👨‍💼) working for a secret organization to monitor and regulate extraterrestrial activity on Earth (👽). The agents often use a memory-erasing pen (🖊️) to maintain secrecy.

3. 🐢🐀👊🍕: "Teenage Mutant Ninja Turtles" - This film centers around four mutant turtles (🐢) trained in ninjutsu by their rat sensei (🐀). They use their martial arts skills (👊) to fight crime, including the notorious villain Shredder. The turtles are known for their love of pizza (🍕).
